config/image_kinds.yml

```diff
@@ -0,0 +1,29 @@
+---
+default:
+  display_name: "Default (960 by 640)"
+  valid_width: 960
+  valid_height: 640
+  versions:
```

Comment:
Yeah, it's already inconsistent sadly. When we're talking about files, Carrierwave calls them [versions](https://github.com/alphagov/whitehall/blob/main/app/uploaders/image_uploader.rb#L13), but when we're talking about assets we call them variants.

I went with versions because the main place this bit of config is used is in a loop to call the Carrierwave `version` method, so it felt better to be consistent with that than with the assets code (that's mostly on the far side of Carrierwave).